tests/catch/unit/stream/hipStreamValue.cc

```diff
@@ -512,7 +512,44 @@ TEST_CASE("Unit_hipStreamValue_Negative_InvalidMemory") {
   HIP_CHECK(hipStreamDestroy(stream));
 }
 
-TEMPLATE_TEST_CASE("Unit_hipStreamWaitValue_Negative_InvalidFlag", "", uint32_t, uint64_t) {
+TEMPLATE_TEST_CASE("Unit_hipStreamValue_Negative_UninitializedStream", "", uint32_t, uint64_t) {
+#if HT_AMD
```

Comment:
Can we remove this and just not have the test run (using json)?

Reply:
@chrispaquot This approach was used because it was the way already implemented tests have been disabled. I don't have access to the jira defect, so I don't know if its content affected the choice of the disable method. If you have more information about the defect and could let me know if it still active, I can switch the json config file instead.

Reply:
@mangupta can you help here?

Reply:
I believe EXSWCPHIPT-96 was resolved via https://github.com/rocm-developer-tools/hipamd/commit/45a715c877391460fd02eebd2ebf27ab80b4a6a8. So we should be able to enable these tests on AMD as well.

Reply:
@mangupta I have rebased the changes, however now when the tests have been enabled on AMD some of them fail and defects will have to be created

Reply:
@nives-vukovic : I don't see any failures with your updated patch. Can you please cross-check once more?

Reply:
@mangupta I have run the tests once more, and several negative tests fail. However, we are currently only able to test it using rocm-5.2.3, so it is possible that these issues have been solved on develop.